meeting-notes/2024-06-13.md

```diff
@@ -0,0 +1,53 @@
+# 2024-06-13
+
+## Agenda
+
+- Assign roles
+- Announcements
+  - Highlight any upcoming milestones that need our attention
+- Agree on agenda
+- Any blockers for Web5 & Rust WG Charters (*Frank*)
+  - **Goal**: Identify if there are any remaining blockers
+  - **Proposal**: Merge on or before Friday, June 14th if no blockers.
+- WG kick offs, timelines, and coordination (*Gabe*)
+  - **Goal**: *TODO: Gabe to define during meeting*
+- Reassign Kotlin DRI (*Jiyoon*)
+  - **Goal**: Callout that the Kotlin SDK DRI should be changed.
+- Call for clarifications
+
+## Attendance
+
+TSC: Frank, Gabe, Henry, Jiyoon, Kendall, Kirah
+
+Observers: None
+
+## Notes
+
+- Assign roles
+  - Facilitator: Henry
+  - Note taker: Frank
+- Announcements
+  - By end of next week, the plan is to validate that a Rust binding for Kotlin will work in a Block
+    container services environment. Jiyoon raised several questions about blockers and the approach
+    being taken.  Kendall and Diane can reach out to Jiyoon if there are any Kotlin questions she
+    can help with.
+- Any blockers for Web5 & Rust WG Charters
+  - Web5 and Rust WG charters have been open for a week.
```

Comment:
and the tbdex charter too https://github.com/TBD54566975/technical-steering-committee/pull/13